src/ast/ast_validations.c

```diff
@@ -64,6 +64,43 @@ static bool _ValidateAllShortestPaths
 	return false;
 }
 
+// validate that shortestPaths is in a supported place
+static bool _ValidateShortestPaths
```

Comment:
Seems like this function would traverse the entire AST, is that correct ?

Reply:
yes it is a copy of the function above _ValidateAllShortestPaths